twitter/API/APIManager.m

```diff
@@ -7,13 +7,17 @@
 //
 
 #import "APIManager.h"
+#import "Tweet.h"
 
 static NSString * const baseURLString = @"https://api.twitter.com";
-static NSString * const consumerKey = // Enter your consumer key here
-static NSString * const consumerSecret = // Enter your consumer secret here
+static NSString * const consumerKey =
+@"NHYEgJ9EhWNiuzIY6Jegxe17n";
+static NSString * const consumerSecret =@"uCDQgYivwUJcQyiJdo5qB7gVtbR8CHMIo8bElUFAQgiGfZZL7R";
```

Comment:
I think I saw something about a better way to handle secret keys but maybe I'm lying.

Reply:
Omg I forgot. i will handle it in the next push